src/gui/programupdater.cpp

```diff
@@ -73,10 +66,17 @@ void ProgramUpdater::checkForUpdates()
 void ProgramUpdater::rssDownloadFinished(const QString &url, const QByteArray &data)
 {
     Q_UNUSED(url);
-
     qDebug("Finished downloading the new qBittorrent updates RSS");
-    QString version;
 
+#ifdef Q_OS_MAC
+    const QString OS_TYPE {"Mac OS X"};
+#elif defined(Q_OS_WIN)
+    const QString OS_TYPE {((QSysInfo::windowsVersion() >= QSysInfo::WV_WINDOWS7)
```

Comment:
This function seems deprecated/obsolete in Qt5

Reply:
pity that Qt deprecated it... looking at the alternatives:
1. QSysInfo::productVersion(), we will have to chop, convert to int and compare.
2. Windows GetVersionEx(), more code required.

Or just stay with QSysInfo::windowsVersion(), I'll pick this one until Qt really removes it.
And I didn't found one in boost.

Reply:
I was thinking pretty much the same. IMO, leave it as is.